src/main/java/org/openpnp/machine/reference/feeder/ReferenceAutoFeeder.java

```diff
@@ -86,6 +86,9 @@ public void feed(Nozzle nozzle) throws Exception {
     
     @Override
     public void postPick(Nozzle nozzle) throws Exception {
+        if (postPickActuatorName == null || postPickActuatorName.equals("")) {

```

Comment:
Not needed.
https://github.com/openpnp/openpnp/blob/d6c8ce67ffc824c00d1f1a69784b4d679c579ea2/src/main/java/org/openpnp/spi/base/AbstractHead.java#L104-L110
_Mark

Reply:
Sorry my mistake: the check is for later.
_Mark